### DISCUSSION TITLE
linux/ubuntu support

### DISCUSSION THREAD

**Discussion by jonathandickinson** [Chia-Network/chia-blockchain#7641] [Ideas]
Hi! Please make this farmable on Linux!

**Comment by joelrb** [comment]
Go to install page to get the Linux flavor you want.

**Comment by keliew** [comment]
you're a funny guy...

**Comment by emlowe** [comment]
There are installers for Ubuntu, Ubuntu (ARM), and RedHat/Centos on https://www.chia.net/download/
